docker/ubuntu20.04-sydr-fuzz/Dockerfile

```diff
@@ -33,4 +33,24 @@ RUN git clone https://github.com/llvm/llvm-project.git && cd llvm-project && \
 RUN echo /usr/lib/x86_64-unknown-linux-gnu > /etc/ld.so.conf.d/libc++.conf && \
     ldconfig
 
+RUN apt-get -y install lcov cargo pkg-config libglib2.0-dev \
```

Comment:
Better install everything in upper apt-get install command

Reply:
You may install rust (cargo) the following way:

```
# Install Rust.
RUN curl https://sh.rustup.rs -o rustup.sh && chmod +x rustup.sh && \
    ./rustup.sh -y && rm rustup.sh

ENV PATH="/root/.cargo/bin:${PATH}"
```

Reply:
Then remove these lines from image-rs Dockerfile and verify that it works.

Reply:
Done